ui/src/sources/components/WelcomeStep.tsx

```diff
@@ -0,0 +1,40 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+
+// Components
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+// Actions
+import {notify as notifyAction} from 'src/shared/actions/notifications'
+
+interface Props {
+  notify: typeof notifyAction
+}
+
+@ErrorHandling
+class SourceStep extends PureComponent<Props> {
+  constructor(props: Props) {
+    super(props)
+  }
+
+  public next = async () => {
+    // navigate to newly created source?
```

Comment:
should something be happening here? or is this part tbd?

Reply:
I think this was tbd, it's a note from Deniz